acceptance/framework/consul/cli_cluster.go

```diff
@@ -200,9 +200,14 @@ func (c *CLICluster) Destroy(t *testing.T) {
 	require.NoError(t, err)
 }
 
-func (c *CLICluster) SetupConsulClient(t *testing.T, secure bool) (*api.Client, string) {
+func (c *CLICluster) SetupConsulClient(t *testing.T, secure bool, release ...string) (*api.Client, string) {
 	t.Helper()
 
+	releaseName := c.releaseName
```

Comment:
what does release name signify here? I thought it would be something like a specific consul release, but I see we're using a random name in the tests you added

Reply:
It's the helm release (i.e. installation) name. The reason that this is here is because in this external servers test (that follows the other external server tests) we spin up two helm installations, one with only a Consul instance that acts as the "external" -- its ACL token that we need for talking to Consul gets persisted in a secret that doesn't match the expected name of the secret used when initializing the Consul client -- so this overrides that.